gcloud/exceptions.py

```diff
@@ -171,6 +172,9 @@ def make_exception(response, content, use_json=True):
     :rtype: instance of :class:`GCloudError`, or a concrete subclass.
     :returns: Exception specific to the error response.
     """
+    if six.PY3 and isinstance(content, bytes):
+        content = content.decode('utf-8')  # pragma: NO COVER Py3K
```

Comment:
It's better to deal with the issue than introducing a pragma no cover here.

Also, unit tests help sort this out.